web/src/main/java/cz/muni/fi/pa165/controllers/ManufacturerController.java

```diff
@@ -0,0 +1,67 @@
+package cz.muni.fi.pa165.controllers;
+
+import cz.muni.fi.pa165.dto.BottleDTO;
+import cz.muni.fi.pa165.dto.ManufacturerDTO;
+import cz.muni.fi.pa165.exceptions.ResourceNotFound;
+import cz.muni.fi.pa165.facade.BottleFacade;
+import cz.muni.fi.pa165.facade.ManufacturerFacade;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.inject.Inject;
+import java.util.List;
+
+/**
+ * @author Martin Sumera
+ */
+@RestController
+@RequestMapping("/manufacturer")
+public class ManufacturerController {
+
+    @Inject
+    private ManufacturerFacade manufacturerFacade;
+
+    @Inject
+    private BottleFacade bottleFacade;
+
+    @RequestMapping(method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
+    public final List<ManufacturerDTO> getManufacturers() {
+        return manufacturerFacade.findAll();
+    }
+
+    @RequestMapping(value = "/{id}", method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
+    public final ManufacturerDTO getManufacturerById(@PathVariable("id") long id) {
+        ManufacturerDTO result = manufacturerFacade.findById(id);
+        if (result != null) {
+            return result;
+        } else {
+            throw new ResourceNotFound();
+        }
+    }
+
+    @RequestMapping(value = "/{id}/production/toxic", method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
+    public final List<BottleDTO> getManufacturerToxicProduction(@PathVariable("id") long id) {
```

Comment:
Toxic a NonToxic sa daju zjednodusit do jednej metody, s tym ze budes z url vytahovat true false a posielat to do metody napr. getAllBottlesFromManufacturerByToxicity, nemusis mat zbytocne dve metody ked staci jedna.